### DISCUSSION TITLE
Suggestions for VI music app

### DISCUSSION THREAD

**Discussion by Energetic-Plasma** [community/community#121248] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I wanted to give 4 suggestions for VI music app :-

1) Add a widget system so we will be able to get player controls from the home screen. 
 
A) Add a CD player animation in the widget so add the music plays it starts rotating. It will look simple classy and OP. 

2) If possible please add a (music visualizer) so it will make a vibe during the music and also makes it look cool. 

3) It would be really helpful if u had ur own 10-band equalizer in the app itself coz many Android devices lack this feature. It will be ok if u don't give any presets. 

4) Also just like Spotify where when a song is added to playlist it automatically detect the song and recommends few songs of same vibe just below it. If it is possible to add this feature in the VI music app plz go forward. 

If u have made it till last then a big thanks to u plz mind my suggestions I really hope u implement some of these features in the VI music app in the future. Thanks a lot for creating such a great music app 🙏🙏🙏

**Comment by ebndev** [comment]
Hi @Energetic-Plasma,

Thanks for posting in the GitHub community! 

It looks like you're trying to leave feedback for the developers of the VI music app so I encourage you to open an issue in their repository or see if they have a support page. While many projects are hosted on Github, GitHub Discussions is a public forum so this may not be the place you intended this message to go.

Hope they see that you're enjoying the app! Best of luck!

**Comment by jamesjohn432** [comment]
Honestly, I feel the same way; these suggestions would make the VI music app way more enjoyable. That last point about auto-recommendations really reminded me why I stick [with Spotify Premium APK free](https://spotifyfree.net/), it nails that vibe-matching feature effortlessly.
